packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-function-settings.tsx

```diff
@@ -80,6 +81,30 @@ const LockedFunctionSettings = (props: Props) => {
         ]);
     }, [domain]);
 
+    function getPrepopulatedAriaLabel() {
+        let str = `Function with equation ${equationPrefix}${equation}`;
+
```

Comment:
Should there be an additional check to validate that there is text added to the labels? Or make the text a required field if the button to add a visible label is clicked?

Reply:
I did think about that, and that is a good point. I do agree with you. Unfortunately, validation in the content editor has been deprioritized in favor of just trusting the content authors and giving them ultimate freedom. The content authors should definitely be checking that the label makes sense before saving this on the exercise.

If this ends up being an issue and content authors ask for extra validation, we would definitely add that in the future.